adafruit_lsm303.py

```diff
@@ -237,6 +261,52 @@ def mag_rate(self, value):
         reg_m = ((value & 0x07) << 2) & 0xFF
         self._write_u8(self._mag_device, _REG_MAG_CRA_REG_M, reg_m)
 
+    @property
+    def accel_gain(self):
+        """The accelerometer's gain."""
+        return self._accel_gain
+
+    @accel_gain.setter
+    def accel_gain(self, value):
+        #assert value in (ACCELGAIN_2, ACCELGAIN_4, ACCELGAIN_8, ACCELGAIN_16)
```

Comment:
Why is this line in here? If there is no intention of using the `assert`, it may make sense to remove this line.